konrad/core.py

```diff
@@ -211,7 +211,7 @@ def get_hours_passed(self):
         Returns:
             float: Hours passed since model start.
         """
-        return self.runtime.total_seconds() / 3_600
```

Comment:
Actually this wasn't a typo ;) Since Python 3.6 it is possible to add underscores in numerical literals to increase readability, e.g. `1_023_342_432`.

I agree that one can argue about whether this is needed for for `3600` ;)

Reply:
Okay, I did not know that, but actually it's quite nice for really long numbers!